Extensions/quick_tags.js

```diff
@@ -273,6 +279,18 @@ XKit.extensions.quick_tags = new Object({
 
 	},
 
+	render_add_bundle_button: function(for_one_click) {
+		var m_add_button = "";
+
+		if (user_tag_array.length >= 30 && 
+			for_one_click === true &&
+			XKit.extensions.quick_tags.preferences.hide_new_bundle_button.value === false) {
```

Comment:
This if condition seems reversed... Do you really want to only be adding the html if we have 30 or more tags, for_one_click is true and hide_new_bundle_button is false?